runtime/jvmti/jvmtiHook.c

```diff
@@ -151,6 +151,10 @@ static void jvmtiHookVirtualThreadDestroy (J9HookInterface **hook, UDATA eventNu
 static void jvmtiHookVirtualThreadMount (J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData);
 static void jvmtiHookVirtualThreadUnmount (J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData);
 #endif /* JAVA_SPEC_VERSION >= 19 */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static void jvmtiHookVMCheckpoint(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData);
```

Comment:
It is desirable for the pointer symbol (`*`) to stay with the argument (at multiple places) rather then the type. e.g. `J9HookInterface **hook`.